modules/troubleshoot/partials/inspect-bundle.adoc

```diff
@@ -0,0 +1,541 @@
+== Inspect the debug bundle
```

Comment:
praise/question: This is great!  Is this new?  @JFlath may have some good info for this as well if you haven't already talked to him.

Reply:
High level thoughts (happy to add more detail if needed):

* Review Redpanda's logs (in `redpanda.log` for non-K8s, `logs/` for K8s)
* Review topic definitions and configs (in `kafka.json`, but this requires a fair bit of `jq` magic)